### PR TITLE
[4.1.0] Changing default tiers

### PR DESCRIPTION
**Pull request by vishmi49**
This PR is raised update details on changing the default rate limiting tiers.
add-default-tier-configurations

### CODE REVIEW

File: en/docs/design/rate-limiting/setting-throttling-limits.md [hunk 1]
```diff
@@ -155,3 +155,21 @@ It is also possible to specify a bandwidth per unit time instead of a number of
...
+
+#### Changing the Default Tiers
+
+Users can change the default tiers by appending the following config to the advanced Configurations in Admin portal.
+
+1.  Log in to the API Manager's Admin console ( `https://localhost:9443/admin` ) and go to the **Settings &gt; Advanced** menu.
+
+    ![](../../assets/img/design/rate-limiting/change-default-tiers-menu.png)
+
+3.  Append the following configurations to the **Advanced Configurations** as required.
```
Comment (tharikaGitHub) [inline en/docs/design/rate-limiting/setting-throttling-limits.md:167]:
```suggestion
2.  Append the following configurations to the **Advanced Configurations** as required.
```

File: en/docs/design/rate-limiting/setting-throttling-limits.md [hunk 2]
```diff
@@ -155,3 +155,21 @@ It is also possible to specify a bandwidth per unit time instead of a number of
             x-wso2-throttling-tier: 20kPerMin
     ```
 </div>
+
+#### Changing the Default Tiers
+
+Users can change the default tiers by appending the following config to the advanced Configurations in Admin portal.
+
+1.  Log in to the API Manager's Admin console ( `https://localhost:9443/admin` ) and go to the **Settings &gt; Advanced** menu.
```
Comment (tharikaGitHub) [inline en/docs/design/rate-limiting/setting-throttling-limits.md:163]:
```suggestion
1.  Log in to the API Manager's Admin Portal ( `https://localhost:9443/admin` ) and go to the **Settings &gt; Advanced** menu.
```

File: en/docs/design/rate-limiting/setting-throttling-limits.md (hunk 1 above)
Comment (vishmi49) [reply en/docs/design/rate-limiting/setting-throttling-limits.md]: Resolved with latest commit

File: en/docs/design/rate-limiting/setting-throttling-limits.md (hunk 2 above)
Comment (vishmi49) [reply en/docs/design/rate-limiting/setting-throttling-limits.md]: Resolved with latest commit

**Review by tharikaGitHub — APPROVED** [review]

**Pull request merged**